packages/client/src/__tests__/integration/happy/rejectNotFound/test.ts

```diff
@@ -50,21 +50,42 @@ for (const constructorKey of Object.keys(cases.constructor)) {
       const value = currentMethod[valueKey]
       test(`rejectOnNotFound | constructor=${constructorKey} | ${method}=${value}`, async () => {
         // It should fail or not
-        expect.assertions(1)
         const PrismaClient = await getTestClient()
         const prisma = new PrismaClient({
           rejectOnNotFound: constructor,
         })
 
-        // Test Rejection
-        try {
+        // This function name is important cause we're searching
+        // for it's name in the stack bellow
+        const testRejectionOnNotFound = async () => {
           const r = await prisma.user[method]({
             where: { id: 'none' },
             rejectOnNotFound: value,
           })
-          expect(r).toMatchSnapshot()
```

Comment:
@YassinEldeeb *suggestion*: I would suggest we keep the existing snapshots in place. In your branch the expect uses the thrown `error.message` to build the assertion which makes the test less likely to break when the error message changes.  In that case we want the test to fail-error messages should stay the same as some people use them to recognize which error was thrown.

Reply:
the snapshot looks like this

```
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rejectOnNotFound | constructor=customError | findFirst=false 1`] = `null`;

exports[`rejectOnNotFound | constructor=customError | findUnique=false 1`] = `null`;

exports[`rejectOnNotFound | constructor=customErrorPerActionPerModel | findFirst=false 1`] = `null`;

exports[`rejectOnNotFound | constructor=customErrorPerActionPerModel | findUnique=false 1`] = `null`;

exports[`rejectOnNotFound | constructor=errorPerAction | findFirst=false 1`] = `null`;

exports[`rejectOnNotFound | constructor=errorPerAction | findUnique=false 1`] = `null`;

exports[`rejectOnNotFound | constructor=false | findFirst=false 1`] = `null`;

exports[`rejectOnNotFound | constructor=false | findFirst=undefined 1`] = `null`;

...
```

so what's the point of it if it's gonna always be equal to null

Reply:
checking for null is the same as checking if it's equal to the snapshot in this case

Reply:
oh sorry wrong line. Will suggest another one.

Reply:
@YassinEldeeb it doesn't look right, we're basically losing a bunch of tests in this case. These should probably be made snapshot tests for `error.message` then.

Reply:
@aqrln  this is a sample from the actual snapshot, but they're all equal to null anyways

Reply:
@YassinEldeeb yes this line doesn't need to be retained :100: , the other one does though